lib/existing-eks-opensource-observability-pattern/index.ts

```diff
@@ -77,10 +78,24 @@ export default class ExistingEksOpenSourceobservabilityPattern {
                 logRetentionDays: 30
             }),
             new blueprints.addons.XrayAdotAddOn(),
-            new blueprints.addons.FluxCDAddOn({"repositories": [fluxRepository]}),
+            new blueprints.addons.FluxCDAddOn({ "repositories": [fluxRepository] }),
             new GrafanaOperatorSecretAddon(),
         ];
 
+        if (utils.valueFromContext(scope, "nginx.pattern.enabled", false)) {
+            ampAddOnProps.openTelemetryCollector = {
+                manifestPath: __dirname + '/../common/resources/otel-collector-config.yml',
+                manifestParameterMap: {
+                    javaScrapeSampleLimit: 1000,
+                    javaPrometheusMetricsEndpoint: "/metrics"
+                }
+            };
+            ampAddOnProps.ampRules?.ruleFilePaths.push(
+                __dirname + '/../common/resources/amp-config/nginx/alerting-rules.yml'
+            );
+            addOns.push(new NginxPrometheusAddon());
```

Comment:
We should not do this, we should rather use multi flux paths Kustomization. Check the API Server PR for more details.

Reply:
remove nginxprometheusaddon and simplified